src/cmd-buildupload

```diff
@@ -48,7 +48,12 @@ def cmd_upload_s3(args):
         if args.build == 'latest':
             args.build = builds.get_latest()
         print(f"Targeting build: {args.build}")
-        s3_upload_build(args, builds.get_build_dir(args.build), args.build)
+        if builds.is_legacy():
```

Comment:
how long are we going to carry this?

Reply:
At least until the RHCOS bucket migrates to the new layout as well.